infra/marin-cluster-template.yaml

```diff
@@ -20,7 +20,7 @@ provider:
 
 
 docker:
-    image: "{{REGION}}-docker.pkg.dev/hai-gcp-models/marin/{% if VLLM %}marin_vllm{% else %}marin_cluster{% endif %}:{{DOCKER_TAG}}"
```

Comment:
nit: should this just always be IMAGE_NAME (or hardcoded as marin-cluster), given we have the separate vllm template anyway?